### ISSUE TITLE
代码状态

### ISSUE THREAD

**Issue by xiran2018** [QiangZiBro/stacked_capsule_autoencoders.pytorch#1]
请问作者，代码完成到什么程度了？是否已经可以跑了？

**Comment by QiangZiBro** [comment]
Thanks for watching, the first experiment (CCAE) has been implemented, you can refer to README for how to run. But the result is not good yet, I will figure out why when I am free. Welcome you to join the discussion.

**Comment by Aiden-song** [comment]
Looking forward to your next commit.

**Comment by Aiden-song** [comment]
> Thanks for watching, the first experiment (CCAE) has been implemented, you can refer to README for how to run. But the result is not good yet, I will figure out why when I am free. Welcome you to join the discussion.

Looking forward to your next commit.

**Comment by superwars** [comment]
请问这个项目是否还在更新？：）

**Comment by keneathhui** [comment]
How to fix the loss=NaN crash after training for a while
